store/src/java/com/zimbra/cs/redolog/op/ModifyVolume.java

```diff
@@ -102,6 +112,9 @@ public void redo() throws Exception {
                 .setMboxGroupBits(mboxGroupBits).setMboxBit(mboxBits)
                 .setFileGroupBits(fileGroupBits).setFileBits(fileBits)
                 .setCompressBlobs(compressBlobs).setCompressionThreshold(compressionThreshold).build();
+        VolumeInfo volInfo = vol.toJAXB();;
+        ExternalVolumeInfoHandler extVolInfoHandler = new ExternalVolumeInfoHandler(Provisioning.getInstance());
```

Comment:
Check for this as well, do we need to override it here?

Reply:
We dont need to overwite JSON backup files (prev, curr versions) as the modifyServerProperties function is taking care for that,
 Infact the architecture is such that where-ever we are setting LDAP attributes we are taking backup before and after as prev and current version respectively